lib/resolveurl/plugins/streamsb.py

```diff
@@ -28,7 +28,7 @@ class StreamSBResolver(ResolveUrl):
     domains = ["sbembed.com", "sbembed1.com", "sbplay.org", "sbvideo.net", "streamsb.net", "sbplay.one",
                "cloudemb.com", "playersb.com", "tubesb.com", "sbplay1.com", "embedsb.com", "watchsb.com"]
     pattern = r'(?://|\.)((?:watch|embed|tube|player|cloudemb|stream)?s?b?(?:embed\d?|play\d?|video)?\.' \
-              r'(?:com|net|org|one))/(?:embed-|e|play|d)?/?([0-9a-zA-Z]+)'
+              r'(?:com|net|org|one))/(?:embed-|e/|play/|d/)?([0-9a-zA-Z]+)'
```

Comment:
The old pattern was fine, any embed not working with the old pattern?

Reply:
Hi @Gujal00 . See when i made adjustments a couple months back to match hxxp://{host}/{media_id}.html, didn't think about if the media_id started with an e or d. To see what im talking about, try this hxxps://watchsb.com/eav4k9md0tgv.html

Reply:
Perfect, Thanks